legacy/reqcounter/reqcounter.go

```diff
@@ -47,16 +48,25 @@ func (rc *RequestCounter) Flush() {
 	rc.Mutex.Lock()
 	defer rc.Mutex.Unlock()
 
-	// Log the number of requests within this measurement window.
-	msg := fmt.Sprintf("From %s to %s [%d min] there have been %d requests to GCR.", rc.Since.Format(TimestampFormat), Clock.Now().Format(TimestampFormat), rc.Interval/time.Minute, rc.Requests)
-	Debug(msg)
+	rc.log()
 
 	if rc.Resettable {
 		// Reset the request counter.
 		rc.reset()
 	}
 }
 
+// log the number of HTTP requests found. If the number of requests exceeds the
+// threshold, an additional warning message will be logged.
```

Comment:
Use the present tense. It's simpler.

`an additional warning message will be logged` -> `log an additional warning message`

Reply:
Ack.